.buildkite/Makefile

```diff
@@ -0,0 +1,5 @@
+# Builds all dhall entrypoints
+check:
+	dhall <<< './src/Prepare.dhall'
+	dhall <<< './src/Monorepo.dhall'
+	ls src/jobs | xargs -I{} bash -c 'dhall <<< ./src/jobs/{}/Pipeline.dhall'
```

Comment:
Suggest: `for f in ./src/lib/**/Pipeline.dhall; do dhall --file "$f"; done`
Xargs is cool, but for wildcarding, this is to read and maintain. If you are married to xargs, still prefer the following due to X-platform compat: `echo ./src/jobs/**/Pipeline.dhall | xargs -n1 -I{} bash -c 'dhall --file "{}"'`